lib/charms/alertmanager_k8s/v0/alertmanager_remote_configurer.py

```diff
@@ -0,0 +1,1888 @@
+                    "default": True,
+                },
+                "api_secret": {
+                    "description": "The API key to use when talking to the WeChat API. Defaults to global.wechat_api_secret.",  # noqa: E501
+                    "type": "string",
+                },
+                "api_url": {
+                    "description": "The WeChat API URL. Defaults to global.wechat_api_url.",
+                    "type": "string",
+                    "format": "uri-reference",
+                },
+                "corp_id": {
+                    "description": "The corp id for authentication. Defaults to global.wechat_api_corp_id.",  # noqa: E501
+                    "type": "string",
+                },
+                "message": {
+                    "description": "API request data as defined by the WeChat API.",
+                    "$ref": "#/definitions/string_or_template_ref_value",
+                    "default": '{{ template "wechat.default.message" . }}',
+                },
+                "message_type": {
+                    "description": "Type of the message type, supported values are `text` and `markdown`.",  # noqa: E501
+                    "type": "string",
+                    "enum": ["text", "markdown"],
+                    "default": "text",
+                },
+                "agent_id": {
+                    "$ref": "#/definitions/string_or_template_ref_value",
+                    "default": '{{ template "wechat.default.agent_id" . }}',
+                },
+                "to_user": {
+                    "$ref": "#/definitions/string_or_template_ref_value",
+                    "default": '{{ template "wechat.default.to_user" . }}',
+                },
+                "to_party": {
+                    "$ref": "#/definitions/string_or_template_ref_value",
+                    "default": '{{ template "wechat.default.to_party" . }}',
+                },
+                "to_tag": {
+                    "$ref": "#/definitions/string_or_template_ref_value",
+                    "default": '{{ template "wechat.default.to_tag" . }}',
+                },
+            },
+            "additionalProperties": False,
+        },
+        "weekdays": {
+            "description": "Days should be specified by name (e.g. 'Sunday'). For convenience, ranges are also accepted of the form : and are inclusive on both ends. Example: 'tuesday:Saturday' or 'Monday'.",  # noqa: E501
+            "type": "string",
+            "pattern": "^([Mm]onday|[Tt]uesday|[Ww]ednesday|[Tt]hursday|[Ff]riday|[Ss]aturday|[Ss]unday)(:($1))?$",  # noqa: E501
+        },
+        "years": {
+            "description": "A numerical value of years. Ranges are accepted. Ranges are inclusive on both ends. Example: '2020:2022' or '2030'.",  # noqa: E501
+            "type": "string",
+            "pattern": "^(2\\d\\d\\d)(:[2-9]\\d\\d\\d)?$",
+        },
+    },
+}
+DEFAULT_RELATION_NAME = "remote-configurer"
```

Comment:
Change relation name to `remote-configuration`

Reply:
Done.